### ISSUE TITLE
It is needed to motivate cluster nodes to do their job

### ISSUE THREAD

**Issue by vizotov** [zerocracy/zold#18]
In the current state of the project cluster nodes do not have a reason to stay online and submit transactions.

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by yegor256** [comment]
@vizotov yes, you are right. I believe, the best way is to just charge a fixed fee for each transaction processing, just like Bitcoin does.

**Comment by vizotov** [comment]
If the fee is fixed, there is a possibility that it may not be enough. This will depend on the exchange rate. In the Bitcoin, the calculation of transaction fees is a partially automatic process, and is partially determined by the market. But here, since the currency is just a debt issuance of the issuer, then he must control the fee in order to keep the currency alive.
